### PR TITLE
docs: W3C API status doc and Holder API Overview

### LINKED ISSUE #345: Document W3C Interop APIs

### PR DESCRIPTION
**Pull request by rolsonquadras**
closes #345 

Signed-off-by: Rolson Quadras <rolson.quadras@securekey.com>

### CODE REVIEW

File: docs/vc-rest/w3c_api_impl_status.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# W3C CCG HTTP API Implementation Status
```
Comment (troyronda) [inline docs/vc-rest/w3c_api_impl_status.md:1]: We might need a more generic title (and filename) as this document mixes both W3C CCG __and__ Transmute Extensions.

Comment (rolsonquadras) [reply docs/vc-rest/w3c_api_impl_status.md:1]: any suggestions ?

Comment (troyronda) [reply docs/vc-rest/w3c_api_impl_status.md:1]: maybe `vc_interop_api` or something.

File: docs/vc-rest/w3c_api_impl_status.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+The TrustBloc VC edge service supports multiple W3C CCG HTTP APIs. This page provides the status of the implementation. For detailed API documentation, refer [TrustBloc VC Service Open API Spec](openapi_demo.md).
```
Comment (troyronda) [inline docs/vc-rest/w3c_api_impl_status.md:3]: (and also Transmute API)

File: docs/vc-rest/w3c_api_impl_status.md (hunk 1 above)
Comment (rolsonquadras) [reply docs/vc-rest/w3c_api_impl_status.md]: [Done](https://github.com/trustbloc/edge-service/compare/9d085de3c4f9b8bdb2114e2c7ddcdda507501ce6..fcd5f59e27bafee8e169ab9bdf2ee79bc8731c19#diff-e3e8ccc1e3586fbcfa5cd223b35fcbdbR1),

File: docs/vc-rest/w3c_api_impl_status.md (hunk 2 above)
Comment (rolsonquadras) [reply docs/vc-rest/w3c_api_impl_status.md]: [Done](https://github.com/trustbloc/edge-service/compare/fcd5f59e27bafee8e169ab9bdf2ee79bc8731c19..c78c913a8ef52721263c5eb7ce3b4d2ec4024a41#diff-e3e8ccc1e3586fbcfa5cd223b35fcbdbR3)

**Pull request merged**